json-rpc/src/fuzzing.rs

```diff
@@ -41,10 +41,15 @@ pub fn fuzzer(data: &[u8]) {
 
     let client = Client::new();
```

Comment:
does this do a network query? (if it does, you should try to avoid this as this slows down fuzzers by a lot!)

Reply:
unfortunately unlike grpc, the inputs to the direct backend method is a bit different from the full payload the client would send in (the server processes the payload to some degree), so to ensure the fuzzed data passes through the server I'll keep this for now..

Reply:
I think that's fine! 

Please consider another PR to improve this. Doing IO usually have pretty bad consequences on the fuzzer performances and that means that you are less likely to find bugs this way.

Usual ways to remove IO:

* Mock the IO (see [the noise fuzzer](https://github.com/libra/libra/blob/258297a2e1c134df67824e8a474091f7639b3b37/network/noise/src/noise_fuzzing.rs#L217) for example)
* bypass the IO by passing data directly to the relevant function